src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
+    private String colorFigure;
+
+    public Figure(String nameFigure, String colorFigure) {
+        this.colorFigure = colorFigure;
+    }
```

Comment:
```suggestion
    public Figure(String color) {
        this.color = color;
    }
```
і так зрозуміло що це колір фігури, бо поле знаходить в середині класу Figure. Тому його можна перейменувати на color